RELEASING.md

```diff
@@ -0,0 +1,62 @@
+# Releasing Kubeclient
+
+## Versioning
+Kubeclient release versioning follows [semvar](https://semver.org/).
+At some point in time it is decided to release version x.y.z.
+Prerequisite: A [changelog](CHANGELOG.md) is generated & commited to the release branch.
+
+```bash
+# USERNAME=bob
+RELEASE_BRANCH="master"
+RELEASE_VERSION="x.y.z"
+GIT_REMOTE="origin"
+GIT_UPSTREAM="https://github.com/abonas/kubeclient.git"
+
+#
+# Install the release gem
+#
+gem install gem-release
+
+
+#
+# Checking out the release branch
+#
+git fetch $GIT_REMOTE
+git checkout $GIT_REMOTE/$RELEASE_BRANCH -b $RELEASE_BRANCH
+
+#
+# Preparing to release
+#
+bundle install
+bundle exec rake test rubocop
+
+#
+# Grabbing an authentication token for rubygems.org api
+#
+curl -u $USERNAME https://rubygems.org/api/v1/api_key.yaml > ~/.gem/credentials; chmod 0600 ~/.gem/credentials
+
+cat /home//.gem/credentials
+---
+:rubygems_api_key: ****
+
+#
+# Bump lib/kubeclient/version.rb and push the commit
+#
+gem bump --version $RELEASE_VERSION --push --destination $GIT_REMOTE
+# Merge the version bump pull request
```

Comment:
does above command create a PR?  I think it just pushes commit to master / $RELEASE_BRANCH?

Reply:
Depends on the remote you are using